documentation/SolutionsRegistryFormat.md

```diff
@@ -59,13 +65,26 @@ The `settingsHandlers` block is unique and one of the most important blocks in t
         "options": {
             "filename": "${{environment}.HOME}\\mySettings.ini"
         },
+        "supportedSettings": {
+            "appsettingA": { ... appsettingA metadata ... },
+            "appsettingB": { ... appsettingB metadata ... },
+            ...
+            "appsettingZ": { ... appsettingZ metadata ... }
+        },
         "capabilities": []
     }
 }
 ```
 
 The important thing to notice here is that this solution example has two references to settingsHandler - one XMLHandler which has been given a reference `myconf` and an INIHandler referred to as `otherconf`.
 
+Note also the `supportedSettings` option. This block is used to determine which application specific settings are relevant to the settingshandler, and also serves as location for providing metadata (such as default values, data type, validation information, etc) about the setting. If there are no relevant metadata related to the setting, the empty object (`{}`) should be used as value.
```

Comment:
There's no need to be mysterious - you should explicitly mention that no such metadata is currently supported :)